src/main/resources/resources/logisim/strings/gui/gui_de.properties

```diff
@@ -576,11 +576,11 @@ simTrueCharTitle = Zeichen für True Wert:
 simTrueColTitle = Farbe für True:
 simUnknownCharTitle = Zeichen für unbekante Werte:
 simUnknownColTitle = Farbe für unbekante:
-simWidthErrorBackgroundTitle = Breitenfehler Hintergrundfarbe:
-simWidthErrorCaptionTitle = Breitenfehler Beschriftungsfarbe:
-simWidthErrorHighlightTitle = Breitenfehler Hervorhebungsfarbe:
-simWidthErrorTitle = Breitefehler Farbe:
-# ==> simClockFrequencyTitle =
+# ==> simWidthErrorBackgroundTitle = Breitenfehler Hintergrundfarbe:
+# ==> simWidthErrorCaptionTitle = Breitenfehler Beschriftungsfarbe:
+# ==> simWidthErrorHighlightTitle = Breitenfehler Hervorhebungsfarbe:
+# ==> simWidthErrorTitle = Breitenfehlerfarbe:
+# ==> simClockFrequencyTitle =  Taktfrequens Farbe:
```

Comment:
"Taktfrequenzfarbe", but anyway commented-out for the moment. So, something for later.